examples/kokkos-tutorials/workload/bfs_bottomup.py

```diff
@@ -0,0 +1,164 @@
+from typing import Tuple
+
+import pykokkos as pk
+
+from parse_args import parse_args
+
+@pk.workload
+class Workload:
+    def __init__(self, N: int, M: int):
+        self.N: int = N
+        self.M: int = M
+
+        self.val: pk.View1D[pk.double] = pk.View([N*M], pk.double)
+        self.vis: pk.View1D[pk.double] = pk.View([N*M], pk.double)
+        self.Matrix: pk.View2D[pk.double] = pk.View([N, M], pk.double)
+        self.max_arr: pk.View1D[pk.double] = pk.View([N], pk.double)
+        self.max_arr2D: pk.View2D[pk.double] = pk.View([N, N], pk.double)
+
+        self.timer_result: float = 0
+        self.element: float = N*M
+
+
+        for i in range(N*M):
+            self.val[i] = N+M
```

Comment:
Consider using the view `fill()` method to initialize views. You can apply this to all views in this class.